ci/autoscaler/pipeline.yml

```diff
@@ -19,7 +19,6 @@ anchors:
       operations/enable-nats-tls.yml
       operations/add-extra-plan.yml
       operations/set-release-version.yml
-      operations/enable-metricsforwarder-via-syslog-agent.yml
```

Comment:
Here you can find a successful run of `acceptance-log-cache` bringing back the old configuration how it should be: https://concourse.app-runtime-interfaces.ci.cloudfoundry.org/teams/app-autoscaler/pipelines/app-autoscaler-release-appautoscaler-739-fix-acceptance-tests/jobs/acceptance-log-cache-syslog/builds/1